### PR TITLE
Making silicon great again

### LINKED ISSUE #3301: Cyborg power consumption is too high
oof owwie ouchie my battery cell. Battery lasts not so much.

Strange, but we dont have it there. Issue is around silicon movement and smooth movement as is - drones are affected too.

### PR DESCRIPTION
**Pull request by soslow4u**
Closes #3301 
Adjusted power consumption for silicons, replaced << with to_chat in affected files.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -263,7 +263,7 @@
 	modules.Add(robot_modules) //This is a global list in robot_modules.dm
 	var/decl/security_state/security_state = decls_repository.get_decl(maps_data.security_state)
 	if((crisis && security_state.current_security_level_is_same_or_higher_than(security_state.high_security_level)) || crisis_override) //Leaving this in until it's balanced appropriately.
-		src << "\red Crisis mode active. Combat module available."
+		to_chat(src, "\red Crisis mode active. Combat module available.")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: "\red" and similar tags are outdated. Use spans.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -405,11 +405,11 @@
 	set name = "Self Diagnosis"
 
 	if(!is_component_functioning("diagnosis unit"))
-		src << "\red Your self-diagnosis component isn't functioning."
+		to_chat(src, "\red Your self-diagnosis component isn't functioning.")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 3]
```diff
@@ -405,11 +405,11 @@
...
 
 	var/datum/robot_component/CO = get_component("diagnosis unit")
 	if (!cell_use_power(CO.active_usage))
-		src << "\red Low Power."
+		to_chat(src, "\red Low Power.")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 4]
```diff
@@ -433,10 +433,10 @@
 	var/datum/robot_component/C = components[toggle]
 	if(C.toggled)
 		C.toggled = 0
-		src << "\red You disable [C.name]."
+		to_chat(src, "\red You disable [C.name].")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 5]
```diff
@@ -433,10 +433,10 @@
...
 	else
 		C.toggled = 1
-		src << "\red You enable [C.name]."
+		to_chat(src, "\red You enable [C.name].")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 6]
```diff
@@ -699,36 +699,36 @@
...
+			to_chat(user, "You must close the cover to swipe an ID card.")
 		else
 			if(allowed(usr))
 				locked = !locked
-				user << "You [ locked ? "lock" : "unlock"] [src]'s interface."
+				to_chat(user, "You [ locked ? "lock" : "unlock"] [src]'s interface.")
 				updateicon()
 			else
-				user << "\red Access denied."
+				to_chat(user, "\red Access denied.")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 7]
```diff
@@ -824,7 +824,7 @@
 
 /mob/living/silicon/robot/proc/installed_modules()
 	if(weapon_lock)
-		src << "\red Weapon lock active, unable to use modules! Count:[weaponlock_time]"
+		to_chat(src, "\red Weapon lock active, unable to use modules! Count:[weaponlock_time]")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/robot/robot.dm]: Same here.

**Comment by soslow4u** [comment]
Done with it, boss man

**Pull request merged**